styleguide/_themes/global/scss/modifiers.scss

```diff
@@ -76,6 +76,18 @@ img { max-width: 100%; }
   margin: 0 auto;
 }
 
+// Horizontal rules
+hr {
+  border: 0;
+  border-top: 1px solid $gray-lighter;
+  margin-bottom: $line-height-computed / 4;
+  margin-top:    $line-height-computed / 4;
```

Comment:
Nope, because then there is nothing defined at all for margin, which means its up to the browser, which isnt always consistent. I would rather set a baseline.